Calling/MainPage.xaml.cs

```diff
@@ -580,5 +577,36 @@ private void TeamsCaptions_CaptionsReceived(object sender, TeamsCaptionsReceived
         #endregion
 #endif
         #endregion
+
+        public async Task HandlePushNotificationIncomingCallAsync(string notificationContent)
+        {
+            PushNotificationDetails pnDetails = PushNotificationDetails.Parse(notificationContent);

```

Comment:
Maybe move this line into the clauses of if (this.callAgent != null)

Reply:
Moved inside the if statement